set.mm

```diff
@@ -647997,6 +647997,18 @@ elements and the subgroup containing only the identity ( ~ simpgnsgbid ).
       AUAMRS $.
   $}
 
+  ${
+    prmsimpcyc.1 $e |- B = ( Base ` G ) $.
+    $( The simple groups of prime order are exactly the cyclic groups.  This is
```

Comment:
False: the cyclic group ZZ/6ZZ is not simple (nor of prime order) !  I think you meant: a group of prime order is cyclic if and only if it is simple. (The statement is slighly more general because of the extensible structure machinery, but that is unimportant.)

Reply:
Thanks, fixed!

Reply:
The statement is very weird, a group of prime order is always simple, so instead of
``|- ( ( # ` B ) e. Prime -> ( G e. SimpGrp <-> G e. CycGrp ) )``
one can simply state
``|- ( ( # ` B ) e. Prime -> G e. CycGrp )``

Am I missing anything obvious?

Reply:
> one can simply state
> ``|- ( ( # ` B ) e. Prime -> G e. CycGrp )``

That's already [`prmcyg`](https://us.metamath.org/mpeuni/prmcyg.html) actually. 
Granted, this is a very small addition to @CatsAreFluffy's work, only adding `prmcyg`... just a pretext to move the "Simple Group" section to main. :-)

Reply:
> That's already [prmcyg](https://us.metamath.org/mpeuni/prmcyg.html) actually.

Oh, indeed. In that case I think your theorem is completely useless, sorry! I can't imagine where it can be used, since we have [prmgrpsimpgd](https://us.metamath.org/mpeuni/prmgrpsimpgd.html) and [prmcyg](https://us.metamath.org/mpeuni/prmcyg.html).

I like the idea of moving the section about simple groups into main, but maybe `prmsimpcyc` can stay in your mathbox then.

Reply:
> I think your theorem is completely useless, sorry! I can't imagine where it can be used, since we have [prmgrpsimpgd](https://us.metamath.org/mpeuni/prmgrpsimpgd.html) and [prmcyg](https://us.metamath.org/mpeuni/prmcyg.html).

Aha! Well, that's all it does : combine [prmgrpsimpgd](https://us.metamath.org/mpeuni/prmgrpsimpgd.html) and [prmcyg](https://us.metamath.org/mpeuni/prmcyg.html).

I can keep it in my mathbox, no problem.

Maybe I will merge this PR and launch another with all these changes ("Simple Groups" to main, `prmsimpcyc` to my mathbox)